Wkhtmltopdf.php

```diff
@@ -780,7 +786,7 @@ protected function _render()
      * @param int $mode
      * @param string $filename
      */
-    public function output($mode, $filename)
+    public function output($mode, $filename = 'temp.pdf')
```

Comment:
what's purpose or default _filename_? I strongly disagree & don't think it's necessary.